<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransformWidget</class>
 <widget class="QWidget" name="TransformWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="8,2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Transform</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">margin: 0;</string>
       </property>
       <property name="text">
        <string>Minimize</string>
       </property>
       <property name="class" stdset="0">
        <string>minimize_but</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </item>
     <item>
      <widget class="QWidget" name="TransformsGrid" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,0,1,0,1">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="TranslateL">
          <property name="text">
           <string>Translation:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="TripletController" name="TranslationTriplet" native="true"/>
        </item>
        <item>
         <widget class="QLabel" name="RotationL">
          <property name="text">
           <string>Rotation:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="TripletController" name="RotationTriplet" native="true"/>
        </item>
        <item>
         <widget class="QLabel" name="ScaleL">
          <property name="text">
           <string>Scale:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="TripletController" name="ScaleTriplet" native="true"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TripletController</class>
   <extends>QWidget</extends>
   <header location="global">tripletcontroller.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
